client/tests/unit/applicant-info-1.spec.ts

```diff
@@ -30,6 +30,6 @@ describe('applicant-info-1.vue', () => {
     expect(wrapper.find('#applicant-info-1-form').element).toBeTruthy()
   })
   it('Initially renders the disabled continue button', () => {
-    expect(wrapper.find('#submit-continue-btn').classes().includes('v-btn--disabled')).toBeTruthy()
+    expect(wrapper.find('#submit-continue-btn-false').classes().includes('v-btn--disabled')).toBeTruthy()
```

Comment:
Nice! Unit tests :) 
I'm sure this test is fine, but in the past we found `toBeTruthy()` and `toBeFalsey()` could sometimes be unreliable, and moved to `.toBe(true)` and `.toBe(false)`

Reply:
BTW you can also do
`wrapper.find('#submit-continue-btn-false').classes('v-btn--disabled')...`

Reply:
my IDE has always auto-suggested toBeTruthy and toBeFalsy so I've always used them I can't say I was specifically even aware of alternative.  But given the issues you say you've experienced I guess I should probably change all uses throughout all the tests.

Reply:
It's just a suggestion, i don't think you need to update all the tests 👍